### DISCUSSION TITLE
正式版增加主题功能

### DISCUSSION THREAD

**Discussion by gaowenchao7918** [Meloong-Git/PCL#2553] [提问] [answered]
想要正式版加一个主题功能，然后赞助解锁，不想为了主题功能去用测试版

**Comment by SuoHouDaiShi** [comment]
那你这不就跟**赞助的原理一样吗**🤔
而且如果真是这样 那做测试版**还有什么意义**呢

**Comment by iPodtouch6557** [comment]
正式版原计划会在一个新的主题系统里开放主题色，但那个是不用赞助的。然而那个主题系统现在看来是遥遥无期了……
至于你这个单独赞助解锁隐藏主题我觉得没啥意义，因为这必然会让赞助系统进行更改，可是一个主题色而已为什么要搞这出呢
~另外买一送一它不香吗~

**Comment by WForst-Breeze** [comment, marked as answer]
**2021 年 8 月 20 日** PCL 1 群公告原文：
【主题功能调整预告】
在联机功能制作完成后，PCL2 计划重做大多数隐藏主题。这大概会导致大多数主题需要重新解锁……
因此，所有隐藏主题，包括极客蓝，从现在开始不再对讨论做出任何限制，你可以自由讨论解锁方法，或者不限人数地攻克解密。
极客蓝解密在获取主题后并没有结束……其剩余奖金将由目前完全完成了后续解密的 31 人平分。
主题功能也会在重做后对正式版开放使用，赞助解锁的功能将在此后改为启动器左上角的标题自定义。
如果你在主题功能重做前解锁了至少 7 个隐藏主题，在重做后会获得一份独特的绝版奖励！

---

而事实是，PCL2 还有很多功能需要不断完善和制作，这件事情的优先级自然就很低，可能一时半会是出不来的。
